backend/attachRendererFiber.js

```diff
@@ -215,6 +216,13 @@ function attachRendererFiber(hook: Hook, rid: string, renderer: ReactRenderer):
     // Suspense
     var isTimedOutSuspense = false;
 
+    // Hooks inspection
+    var containsHooks =
+      (tag === FunctionComponent ||
+        tag === SimpleMemoComponent ||
+        tag == ForwardRef) &&
```

Comment:
Are you wanting to use `==` for the `FowardRef` comparison here?

Reply:
Typo. Doesn't actually matter in this case but I'll fix it.